GoogleForCreators/web-stories-wp
# Dashboard: Update side nav designs

## Summary

Updates the side nav designs. [Figma designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=467%3A13745)

## Relevant Technical Choices

The icons have an "active" and "inactive" state. I added separate svgs for each. Each icon has a slightly different active state so I added them separately. If we want we could use styled components to do the same thing.

## To-do

- Change text in side nav links. These are connected to karma tests and can be annoying so I'll do those separately.
- Found a bug where re-sizing the dashboard when visiting the editor settings won't collapse the side nav. Will fix this in a separate pr.

## User-facing changes

- remove side nav right border
- add icons (active + inactive states)
- move `support` and `settings` links to the bottom of the side nav
- change `Editor Settings` link to be just `Settings`
- remove font weight `bold` from the nav link active state

## Testing Instructions

- go to dashboard
- click side nav links to make sure the active states look correct
- resize the window to make sure side nav collapsing still works correctly
- should be keyboard accessible and screen reader accessible

|Interacting with links|Collapsing side nav|
|--|--|
|![side-nav-1](https://user-images.githubusercontent.com/22185279/104767662-c5476c00-5729-11eb-8cb4-f5ef4d648c93.gif)|![side-nav-collapse](https://user-images.githubusercontent.com/22185279/104767672-c9738980-5729-11eb-8f51-d9e20860cfa8.gif)|

---



Fixes #5054
